source/opt/CMakeLists.txt

```diff
@@ -66,6 +67,7 @@ add_library(SPIRV-Tools-opt
   decoration_manager.cpp
   def_use_manager.cpp
   dead_branch_elim_pass.cpp
+        dead_variable_elimination.cpp
```

Comment:
spacing is wonky.
more importantly, needs a corresponding update to Android.mk

Reply:
fixed

Reply:
Seems over-indented.  Could it be just a rendering issue on the web client?